### PR TITLE
Enable ActiveRecord

### PR DESCRIPTION
**Pull request by ethax-ross**
### Trello card

[Trello-1514](https://trello.com/c/fnHpXkZt/1514-add-postgres-to-tta-service)

### Context

We require a storage mechanism for the TTA feedback form submissions and our current options aren't a good fit for this feature (GA, Prometheus, Logs).

### Changes proposed in this pull request

- Enable ActiveRecord

Enable ActiveRecord in Rails and configure production database to point to the `postgres` instance in `VCAP_SERVICES`.

Uses a postgres service in the GitHub Action for running tests.

### Guidance to review

Nothing has changed as far as the user is concerned in this PR; as long as the app passes CI and boots in the review space it should be good.

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -53,8 +53,20 @@ jobs:
     services:
       registry:
         image: registry:2
-        ports: 
+        ports:
           - 5000:5000
+      postgres:
+        image: postgres
```
Comment (peteryates) [inline .github/workflows/build.yml:59]: Worth specifying a version?

**Review by peteryates — APPROVED** [review]
Looks fine to me. Left a comment on specifying a postgres version but not sure how important it is here.

File: .github/workflows/build.yml (hunk 1 above)
Comment (ethax-ross) [reply .github/workflows/build.yml:59]: Yeah good shout I think we use v11 I'll get it updated

**Review by leoapost — APPROVED** [review]
Looks good.

**Pull request merged**